packages/route-viewer-overlay/src/index.tsx

```diff
@@ -76,7 +76,7 @@ const RouteViewerOverlay = (props: Props): JSX.Element => {
 
         current?.fitBounds(bounds, {
           duration: 500,
-          padding: 200
+          padding: window.innerWidth > 500 ? 200 : undefined
```

Comment:
Do you think these could become a ratio of, I guess both the width and height of the window?

Reply:
It's possible, but I'd want to debug the issue more. I can't quite figure out what about the map size makes maplibregl fail to set the bounds. The 500 value seems to cover most failures. I don't think screen height affects the bug.